graylog2-server/src/main/java/org/graylog2/rest/resources/cluster/ClusterSystemResource.java

```diff
@@ -116,5 +117,26 @@ public SystemThreadDumpResponse threadDump(@ApiParam(name = "nodeId", value = "T
             throw new WebApplicationException(response.message(), BAD_GATEWAY);
         }
     }
+
+    @GET
+    @Timed
+    @ApiOperation(value = "Get a process buffer dump of the given node")
+    @RequiresPermissions(RestPermissions.PROCESSBUFFER_DUMP)
+    @Path("{nodeId}/processbufferdump")
```

Comment:
What do you think about adding another endpoint that returns the dumps for all nodes? That would make it way easier to use. We have `ProxiedResource#getForAllNodes()` for that. Example:

https://github.com/Graylog2/graylog2-server/blob/cb33786987e845aed42956bed67a335c2b91471c/graylog2-server/src/main/java/org/graylog2/rest/resources/cluster/ClusterSystemJobResource.java#L74

Reply:
Nice. Thanks. BTW, I think we should also add this to get all metrics (`org.`) from the cluster